src/qs_apt_fdiff_types.F

```diff
@@ -0,0 +1,35 @@
+!--------------------------------------------------------------------------------------------------!
+!   CP2K: A general program to perform molecular dynamics simulations                              !
+!   Copyright 2000-2025 CP2K developers group <https://cp2k.org>                                   !
+!                                                                                                  !
+!   SPDX-License-Identifier: GPL-2.0-or-later                                                      !
+!--------------------------------------------------------------------------------------------------!
+
+! **************************************************************************************************
+!> \brief Atomic Polarization Tensor calculation by dF/d(E-field) finite differences
+!> \author Leo Decking, Hossam Elgabarty
+! **************************************************************************************************
+
+MODULE qs_apt_fdiff_types
```

Comment:
Does it make sense to move these definitions into the module qs_apt_fdiff_methods?